tools/ete/ete_genetree_splitter.xml

```diff
@@ -25,6 +26,10 @@ python '$__tool_directory__/ete_genetree_splitter.py'
             <option value="0" selected="true">gene_species</option>
             <option value="1">species_gene</option>
         </param>
+        <param name="splitter" type="select" label="GeneTree splitting algorithm">
+            <option value="dups">Split by Duplication</option>
+            <option value="treeko" selected="true">Split using TreeKO algorithm</option>
```

Comment:
Should this be the new default or do we want to keep the old behaviour?

Reply:
Either are fine

Reply:
I'd keep the old behaviour as default then:
```suggestion
            <option value="treeko">Split using TreeKO algorithm</option>
```